static/js/common.js

```diff
@@ -67,9 +67,7 @@ exports.password_warning = function (password, password_field) {
         return undefined;
     }
 
-    if (password_field) {
```

Comment:
hm, we still need this if statement, I think. E.g. there wouldn't be a password field if email auth is not enabled.